run_test.sh

```diff
@@ -82,18 +98,19 @@ function run_all_tests {
 }
```

Comment:
should we add the screenshot tests here? it'll be nice to have one command to use for verification prior to sending a pr

Reply:
done